Jasmine/Tetris/ViewModels/TetrisViewModel.swift

```diff
@@ -1,3 +1,14 @@
+import Foundation
+
 class TetrisViewModel: TimedGameViewModel {
+    /// Specifies the total time allowed in the game.
+    var totalTimeAllowed: TimeInterval
+
+    /// Specifies the time remaining in the game.
+    var timeRemaining: TimeInterval
 
+    init() {
+        totalTimeAllowed = 0
+        timeRemaining = 0
+    }
```

Comment:
feels kind of clumsy to have to redeclare and initialise them in the main VM class right?
Worse still, they can't be implemented in `extension TetrisViewModel: TimedGameViewModel` so they are polluting the main VM class.
Perhaps better to have it as an object then (option 3)?
At least there will only be 1 variable and a single line of code in init to instantiate the timer class.
@xdrawks @li-kai what do u guys think?